.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
```

Comment:
Can you merge develop into this? Is this file not in your .gitignore?

Reply:
I've fixed this now on the new PR #24